include/secp256k1_musig.h

```diff
@@ -154,6 +161,41 @@ SECP256K1_API int secp256k1_musig_pubkey_combine(
     size_t n_pubkeys
 ) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(3) SECP256K1_ARG_NONNULL(5);
 
+/** Tweak an x-only public key by adding the generator multiplied with tweak32
+ *  to it. The resulting output_pubkey with the given internal_pubkey and tweak
+ *  passes `secp256k1_xonly_pubkey_tweak_test`.
+ *
+ *  This function is only useful before initializing a signing session.
+ *  Otherwise just use `secp256k1_xonly_pubkey_tweak_add`. Can only be called
```

Comment:
Rather than "Otherwise just use" you could say "If you are only computing a public key, but not yet signing with it, you can just use".

As written, I wasn't clear on where in the workflow I was supposed to use each function (though it did become clear after reading a few more doccomments)

Reply:
fixed